atlp-rwanda/phantom-be-elite
# #19 Chore: setup docker

### What does this PR do?
Add the docker config file for creating images of the application. This will be done at the end of the project, at least three images will be created and communication will be established between them.
### Description of Task to be completed?
- To create an image of backend
- To configure and run database in a docker container
- Both docker images can communicate while up and running
### How should this be manually tested?
`Docker` should be `installed` in your pc and configured to create and run images

`Clone the repo`

`checkout` to branch `ch-setup-docker-19
`

pull the changes by `git pull`

After checking that docker app is already installed,  Run `docker-compose up -d` to create an image from the docker-compose file and create container.

Then go to the browser and you should see the message being rendered. `IT IS NO MORE NECESSARY TO START SERVER LOCALLY`, docker handles them in the container

### Any background context you want to provide?
N/A

### What are the relevant pivotal tracker stories?
[Trello Board](https://trello.com/c/6CnMHtYC/19-setup-docker)

Screenshots (if appropriate)

[WIP #19]
